contrib/archlinux/PKGBUILD

```diff
@@ -14,12 +14,18 @@ install=
 source=("$pkgname"::'git+https://github.com/gchaincl/httplab.git')
 md5sums=(SKIP)
 
+prepare() {
+	cd "$pkgname"
+	GOPATH="$srcdir"
```

Comment:
Not sure if this is going to work as expected, the directory structre will be $srcdir/httplab, in order to make it work, we should have $srcdir/github.com/gchaincl/httplab.
I'm I missing something?